README.md

```diff
@@ -1 +1,32 @@
-# FirstReview
\ No newline at end of file
+
+
+With this project I try to play the role of a reviewer.I review my old code from Trials and try to detect
+any bad coding style,solutions and conventions, extended use of div's or not using any semantic tags.
+With the new things I've learned I should find alot of these and try to use the correct tags and
+conventions in the future projects.
+
+
+Built With HTML5, CSS
+
+Live Demo Link
+
+https://simongrchevski.github.io/FirstReview/
```

Comment:
[OPTIONAL] You can format the link to the live preview of the project like this:
```
[Live Demo Link](https://simongrchevski.github.io/FirstReview/)
```
This will make it more readable in the README and make it look even more professional.